romancal/resample/resample_utils.py

```diff
@@ -170,7 +174,7 @@ def build_mask(dqarr, bitvalue):
     ----------
     dqarr : numpy.ndarray
         Input DQ array.
-    bitvalue : int
+    bitvalue : str
         Bitvalue flag.
```

Comment:
Let's also change the doc to be a "bitvalue flag string" or something, and maybe 'see astropy.nddata.bitmask.interpret_bit_flags for details.'